rfcs/0004-releases.md

```diff
@@ -57,9 +57,12 @@ The following steps would need to be taken to implement this from a technical pe
   * Add link to update guide in release tag on GH
   * Release triggers a slack webhook to notify that a new release is ready with link to update guide
 
+## Schedule
+* Tradeoffs exist around releases. Too long of a period between releases can potentially cause friction for teams who need new features quickly and are forced to termporarily use the next branch. On the other hand, too short and we increase "update noise" for teams. Additionally, less frequent releases introduce more risk when updating the Eik aliases.
```

Comment:
> less frequent releases introduce more risk when updating the Eik aliases.

Would patch and minor version changes introduce that risk, too?

If not, would it make sense to differentiate release schedule based on whether it's a major or minor/patch update? Just trying to think how "noisy" it could be to get notified of the latter on a weekly basis. If that was already discussed elsewhere then ignore my comment 😄

Reply:
There's a question, I think, around how valuable it would be to have process around major changes. The best way I've seen to handle this is to add deprecation messages during minor and patch releases and then every so often, do a major by removing deprecated features all in one go. This reduces the frequency of major releases and makes for a very clear upgrade path for users. It would require us to be pretty dedicated to the process, very thorough etc which could be challenging and introduce significant overhead so I'd be a bit reluctant.

Reply:
> > less frequent releases introduce more risk when updating the Eik aliases.
> 
> Would patch and minor version changes introduce that risk, too?

Any changes introduce the risk but small and often is always easier to handle than larger and less often.